src/mumble/MainWindow.cpp

```diff
@@ -198,6 +208,78 @@ MainWindow::MainWindow(QWidget *p)
 					 &PluginManager::on_serverSynchronized);
 
 	QAccessible::installFactory(AccessibleSlider::semanticSliderFactory);
+
+	applyTheme();
+}
+
+void MainWindow::applyTheme() {
```

Comment:
We already have a lot of theme handling logic in here https://github.com/mumble-voip/mumble/blob/70dea6077854d79d4cb60b3ed1e8f1f8f78963a5/src/mumble/Themes.cpp
You should probably reuse that and/or modify the ``Themes.cpp`` file directly. That would probably be preferable over re-implementing your own theme loading logic.